docs/servicenow.md

```diff
@@ -0,0 +1,173 @@
+         item='ServiceNow')
+```
+
+Tested create() ticket fields:
+
+```python
+category = 'Category'
+item = 'ServiceNow'
+contact_type = 'Email'
+opened_for = 'dept'
+assigned_to = 'username'
+impact = '2'
+urgency = '2'
+priority = '2'
+email_from = 'username@domain.com'
+hostname_affected = '127.0.0.1'
+state = 'Work in Progress'
+watch_list = ['username@domain.com', 'user2@domain.com', 'user3@domain.com']
+comments = 'New comment to be added'
+```
+
+### get_ticket_content(self, ticket_id) <a name="content"></a>
+
+Retrieves ticket content as a dictionary. Optional parameter ticket_id
+specifies which ticket should be retrieved this way. If not used, method calls
+for ticket_id provided by ServiceNowTicket constructor (or create method).
+
+```python
+# ticket content of the object t
+t.get_ticket_content()
+# ticket content of the ticket <ticket_id>
+t.get_ticket_content(ticket_id=<ticket_id>)
+```
+
+### edit(self, \*\*kwargs) <a name="edit"></a>
+
+Edits fields in a ServiceNow ticket. Keyword arguments are used to
+specify ticket fields. Most of the fields overwrite existing fields.
+One known exception to that rule is 'comments' which adds new comment when
+specified.
+
+```python
+t.edit(short_description='Ticket summary')
+```
+
+Tested edit() ticket fields:
+
+```python
+category = 'Category'
+item = 'ServiceNow'
+contact_type = 'Email'
+opened_for = 'dept'
+assigned_to = 'username'
+impact = '2'
+urgency = '2'
+priority = '2'
+email_from = 'username@domain.com'
+hostname_affected = '127.0.0.1'
+state = 'Work in Progress'
+watch_list = ['username@domain.com', 'user2@domain.com', 'user3@domain.com']
+comments = 'New comment to be added'
+```
+
+### add_comment(self, comment) <a name="comment"></a>
+
+Adds a comment to a ServiceNow ticket. Note that comments cannot be modified
+or deleted in the current implementation.
+
+```python
+t.add_comment('Test comment')
+```
+
+### change_status(self, status) <a name="status"></a>
+
+Changes status of a ServiceNow ticket.
+
+```python
+t.change_status('Work in Progress')
+```
+
+### add_cc(self, user) <a name="add_cc"></a>
+
+Adds watcher(s) to a ServiceNow ticket. Accepts email addresses in the form
+of list of strings or one string representing one email address.
+
+```python
+t.add_cc('username@domain.com')
+```
+
+### rewrite_cc(self, user) <a name="rewrite_cc"></a>
+
+Rewrites current watcher list in the ServiceNow ticket. Accepts email addresses
+in the form of list of strings or one string representing
+one email address.
+
+```python
+t.rewrite_cc(['username@domain.com', 'user2@domain.com', 'user3@domain.com'])
+```
+
+### remove_cc(self, user) <a name="remove_cc"></a>
+
+Removes users from the current watcher list in the ServiceNow ticket. Accepts
+email addresses in the form of list of strings or one string representing
+one email address.
+
+```python
+t.remove_cc(['username@domain.com', 'user3@domain.com'])
+```
+
+### devops_one_url() <a name="devops_one"></a>
```

Comment:
I still feel like we should remove the devops_one_url() stuff from ticketutil. Because it's a helper function for an internal instance of service now. Do you agree with this?